lib/miq_automation_engine/models/miq_ae_class.rb

```diff
@@ -12,7 +12,8 @@ class MiqAeClass < ApplicationRecord
 
   validates_presence_of   :name, :namespace_id
   validates_uniqueness_of :name, :case_sensitive => false, :scope => :namespace_id
-  validates_format_of     :name, :with => /\A[A-Za-z0-9_.-]+\z/i
+  validates_format_of     :name, :with    => /\A[A-Za-z0-9_.-]+\z/i,
+                                 :message => "Only alpha numeric and _ . - charachters are allowed"
```

Comment:
The `"Only alphanumeric and ..."` string should be inside a `N_()` call, as it's described here:

https://github.com/grosser/gettext_i18n_rails#activerecord---error-messages

and then at the place where this message can / will be propagated into the UI, the message needs
to be wrapped with `_()`.